core/src/navigation/LeftNav.svelte

```diff
@@ -184,6 +185,10 @@
   let store = getContext('store');
   let getTranslation = getContext('getTranslation');
   let addNavHrefForAnchor = false;
+  let btpToolLayout = LuigiConfig.getConfigBooleanValue(
+      'settings.btpToolLayout'
+    ) && GenericHelpers.requestExperimentalFeature('btpToolLayout', false);
```

Comment:
in App.svelte we check if experimental is **TRUE** , and here if it must be *false**. 
Shouldn't it be also TRUE here too, or am I missing something. 
The Left Nav shows up properly when testing with: 
```
settings.btpToolLayout: true
settings.experimental.btpToolLayout: true
```

And doesn't show at all for the following. 
```
settings.btpToolLayout: true
settings.experimental.btpToolLayout: false
```

So I am wondering if here we should have TRUE-TRUE as well 🤔

Reply:
see above, here no warning should be printed on console (one is enough ;) )